docs/20_setup.md

```diff
@@ -75,7 +75,7 @@ sudo mv /tmp/miactl /usr/local/bin
 If you want to run the cli in its environment or you want to test the cli you can use the Docker image:
 
 ```sh
-docker run ghr.io/mia-platform/miactl@0.12.1 miactl
+docker run ghcr.io/mia-platform/miactl:0.12.1 miactl
```

Comment:
Thank you for spotting the error, can I ask you to also fix the wrong tag format?

```suggestion
docker run ghcr.io/mia-platform/miactl:v0.12.1 miactl
```